Sources/XcodeExport/Resources/Color+extension.swift.stencil

```diff
@@ -1,15 +1,52 @@
 {% include "header.stencil" %}
 import SwiftUI
-{% if assetsInSwiftPackage %}
+{% if assetsInSwiftPackage and resourceBundleNames %}
+private class BundleProvider {
+    static let bundle = Bundle._module
+}
+{% elif assetsInSwiftPackage and not resourceBundleNames %}
 private class BundleProvider {
     static let bundle = Bundle.module
 }
-{% elif not assetsInMainBundle %}
+{% elif not assetsInSwiftPackage and not resourceBundleNames %}
 private class BundleProvider {
     static let bundle = Bundle(for: BundleProvider.self)
 }
-
 {% endif %}
 public extension Color {{ "{" }}{% for color in colors %}
     static var {{ color.name }}: Color { Color({% if useNamespace %}"{{ color.originalName }}"{% else %}#function{% endif %}{% if not assetsInMainBundle %}, bundle: BundleProvider.bundle{% endif %}) }{% endfor %}
 }
+{% if resourceBundleNames %}
+private extension Foundation.Bundle {
```

Comment:
You have the same `private extension Foundation.Bundle...` code in four Stencil files. Avoid code duplication by extracting duplicated code in a separate file and use Stencil's `include` keyword.